src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -25,17 +26,25 @@ public void actionPerformed(ActionEvent e) {
 		{
 			DrawerFeature.getDrawerController().clearPanel();
 			FiguresJoe.figureScript1(driverManager.getCurrentDriver());
-		}
-		else if (e.getActionCommand().toString().compareTo("Figure Joe 2") == 0)
+		} else if (e.getActionCommand().toString().compareTo("Figure Joe 2") == 0)
 		{
 			DrawerFeature.getDrawerController().clearPanel();
 			FiguresJoe.figureScript2(driverManager.getCurrentDriver());
-		}
-		else if (e.getActionCommand().toString().compareTo("Figure Jane") == 0)
+		} else if (e.getActionCommand().toString().compareTo("Figure Jane") == 0)
 		{
 			DrawerFeature.getDrawerController().clearPanel();
 			FiguresJane.figureScript(jane);
+		} else if (e.getActionCommand().toString().compareTo("Test1") == 0)
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.